.travis.yml

```diff
@@ -19,7 +19,7 @@ cache:
 
 before_install:
   - nvm install && nvm use
-  - npm install npm@latest -g
+  - npm install npm@5.7.1 -g
```

Comment:
Via npm: _"npm@latest is going to be 5.6.0 for the next week or two. 5.7.1 is released under npm@next -- we stagger all our releases this way, to give early adopters time to report major bad stuff"_